com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/HDRP/Water/ComputeWaterVertexPositionNode.cs

```diff
@@ -0,0 +1,38 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class ComputeWaterVertexPositionNode : IStandardNode
+    {
+        public static string Name => "ComputeWaterVertexPosition";
+        public static int Version => 1;
+
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Name,
+@"Out = GetWaterVertexPosition(positionWS);",
```

Comment:
The P in positionWS needs to be capitalized to match the one in the ParameterDescriptor